content/blog/modern-text-features/index.Rmd

```diff
@@ -0,0 +1,370 @@
+---
+output: hugodown::hugo_document
+
+slug: modern-text-features
+title: Modern Text Features in R
+date: 2021-02-06
+author: Thomas Lin Pedersen
+description: >
+  ragg has taken a major leap forward in text rendering capabilities with the
+  latest releases of systemfonts, textshaping, and ragg itself. This post will
+  go into detail with what is now possible and how it compares to the build in 
+  devices.
+
+photo:
+  url: https://unsplash.com/photos/bMybTSV7RFY
+  author: Natalia Y
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [deep-dive] 
+tags: []
+---
+
+I'm extremely pleased to present some new functionality when it comes to text rendering and font support in R.
+This is the culmination of work that started during the development of the ragg package where I was first exposed to the intricacies of text rendering.
+The new features presented herein spans the systemfonts, textshaping, and ragg packages, but from a user point of view everything will be available simply by using the graphic devices in ragg.
+
+ragg is now in a state where it "just works", and works comparably across all major operating systems. 
+This is a first for a raster device in R and is the result of several years of dedicated work. 
+I couldn't be happier with this release!
+
+The features that will be discussed in the following are:
+
+1.  Support for non-Latin scripts including Right-to-Left (RtL) scripts as well as a mix between script directions (bidirectional text)
+2.  Support for OpenType features such as ligatures, glyph substitutions, etc.
+3.  Support for color fonts
+4.  Support for font fallback
+
+The tl;dr of it is that all area mentioned above now has full support in ragg out of the box, but I'd invite you to read on to learn how it works, how to control it, and what it all means for you as a user.
```

Comment:
I think this would be a good place for a reminder how to use ragg (i.e. how to manually set the graphics device and how to set it as the default in RStudio), and a pointer back to other posts about ragg.